index.d.ts

```diff
@@ -0,0 +1,77 @@
+export interface Options {
+	/**
+	 * @default "-"
+	 *
+	 * @example
+	 *
```

Comment:
I think all the examples should be copy-pastable, meaning we need to include the import statement in each example too.

Reply:
In the readme.md, too?

Reply:
Yes :)